keep-ui/app/incidents/incidents-table.tsx

```diff
@@ -96,7 +99,14 @@ export default function IncidentsTable({
       id: "user_summary",
       header: "Summary",
       cell: ({ row }) => (
-        <div className="text-pretty min-w-96">{row.original.user_summary}</div>
+        <div className="text-pretty min-w-96">
+          <Markdown
+            remarkPlugins={[remarkRehype]}
+            rehypePlugins={[rehypeRaw]}
```

Comment:
What are these 2 used for?

Reply:
`remarkRehype` is to enable rehype support in general and rehypeRaw is to enable raw-html rendering